src/importexport/musicxml/tests/data/testMaxNumberLevel.mscx

```diff
@@ -0,0 +1,3198 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This test is not yet used, or is it?

Reply:
Adding `void maxNumberLevel() { mxmlImportTestRef("testMaxNumberLevel"); }` to https://github.com/musescore/MuseScore/blob/master/src/importexport/musicxml/tests/tst_mxml_io.cpp seems to work and pass